contracts/util/LoanLib.sol

```diff
@@ -347,18 +347,30 @@ library LoanLib {
         @notice Payments are made towards the interest first.
         @param loan The loan the payment is for.
         @param toPay The amount of tokens to pay to the loan.
+        @return principalPaid the amount of principal paid back
+        @return interestPaid the amount of interest paid back
     */
-    function payOff(TellerCommon.Loan storage loan, uint256 toPay) public {
+    function payOff(TellerCommon.Loan storage loan, uint256 toPay)
+        public
+        returns (uint256, uint256)
+    {
+        uint256 interestPaid;
+
         if (toPay >= loan.interestOwed) {
-            toPay = toPay.sub(loan.interestOwed);
+            interestPaid = loan.interestOwed;
+            toPay = toPay.sub(interestPaid);
             loan.interestOwed = 0;
             if (toPay == 0) {
-                return;
+                return (0, interestPaid);
```

Comment:
Suggestion: the logic looks good to me so I'll approve it but I have a small suggestion. The logic is simple so it's not that complex to understand the meaning of this 0 value, but probably we could use a constant to describe the meaning of this 0 returned value if we want to be more descriptive (this also applies for other cases where we use 0 values).